Formula/open-babel.rb

```diff
@@ -1,26 +1,11 @@
 class OpenBabel < Formula
   desc "Chemical toolbox"
-  homepage "http://www.openbabel.org"
-
+  homepage "https://openbabel.org"
+  url "https://downloads.sourceforge.net/project/openbabel/openbabel/2.4.0/openbabel-openbabel-2-4-0.tar.gz"
```

Comment:
yes good idea. the checksum is the same for https://github.com/openbabel/openbabel/archive/openbabel-2-4-0.tar.gz